package.json

```diff
@@ -111,16 +113,26 @@
     "test-ci": "yarn build && yarn test-only",
     "test-only": "node --max_old_space_size=4096 node_modules/jest/bin/jest.js --verbose",
     "test-coverage": "node --max_old_space_size=4096 node_modules/jest/bin/jest.js --coverage --verbose",
-    "watch": "gulp watch"
+    "watch": "gulp watch",
+    "commit": "git-cz"
```

Comment:
nit, alphabetize and put this above?

Reply:
Will do in a follow up to avoid a rebuild for this.